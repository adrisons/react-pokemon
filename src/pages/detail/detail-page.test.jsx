import React from "react";
import ReactDOM from "react-dom";
import { BrowserRouter as Router } from "react-router-dom";
import DetailPage from "./detail-page";

describe("GIVEN: DetailPage", () => {
  it("THEN: it renders without crashing", () => {
    const div = document.createElement("div");
    ReactDOM.render(
      <Router>
        <DetailPage />
      </Router>,
      div
    );
    ReactDOM.unmountComponentAtNode(div);
  });
});
