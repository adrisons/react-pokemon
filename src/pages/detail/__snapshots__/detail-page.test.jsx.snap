// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`GIVEN: DetailPage THEN: should match snapshot 1`] = `
<div
  className="loading"
>
  <img
    alt="loading"
    src="pokeball.png"
  />
</div>
`;
